nvdaHelper/local/rpcSrv.cpp

```diff
@@ -27,7 +27,8 @@ using namespace std;
 typedef RPC_STATUS(RPC_ENTRY *RpcServerRegisterIf3_functype)(RPC_IF_HANDLE,UUID __RPC_FAR*,RPC_MGR_EPV __RPC_FAR*,unsigned int,unsigned int,unsigned int,RPC_IF_CALLBACK_FN __RPC_FAR*,void __RPC_FAR*);
 
 RPC_IF_HANDLE availableInterfaces[]={
-	nvdaController_NvdaController_v1_1_s_ifspec,
+	nvdaController_NvdaController_v1_0_s_ifspec,
+	nvdaController_NvdaController2_v1_0_s_ifspec,

```

Comment:
shouldn't this be v2.0 if thats how it is being described in the readme?

```suggestion
	nvdaController_NvdaController_v2_0_s_ifspec,
```

Reply:
Thoughts on this issue?

Reply:
Agreed with @leonardder here. The specific interface version is somewhat internal and should be 1.0 as it is the first version. the public client can be called version 2, that is fine with me.